src/app/components/ScrollablePromo/PromoList/index.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { useContext } from 'react';
+import styled from '@emotion/styled';
+import { GEL_SPACING, GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+import {
+  GEL_GROUP_0_SCREEN_WIDTH_MIN,
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+import { arrayOf, shape, string, oneOfType, object } from 'prop-types';
+import Promo from '../Promo';
+import { ServiceContext } from '#contexts/ServiceContext';
+
+const ScrollPromo = styled.ul`
+  display: flex;
+  list-style: none;
+  ${({ dir }) => `padding-${dir === 'ltr' ? 'left' : 'right'}: 0;`}
+  margin: 0;
+
+  overflow-x: scroll;
+  /* Avoid using smooth scrolling as it causes accessibility issues */
+  scroll-behavior: auto;
+  -webkit-overflow-scrolling: touch;
+
+  /* Hide scrollbar */
+  scrollbar-width: none;
+  -ms-overflow-style: none;
+  &::-webkit-scrollbar {
+    display: none;
+  }
+`;
+
+const StyledList = styled.li`
+  display: flex;
+  flex-shrink: 0;
+  @media (min-width: ${GEL_GROUP_0_SCREEN_WIDTH_MIN}){
+    margin-${({ dir }) => (dir === 'ltr' ? 'left' : 'right')}: ${GEL_SPACING};
+    &:first-child {
+      margin-${({ dir }) => (dir === 'ltr' ? 'left' : 'right')}: ${GEL_SPACING};
+    }
+    &:last-child {
+      margin-${({ dir }) => (dir === 'ltr' ? 'right' : 'left')}: ${GEL_SPACING};
+    }
+  }
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}){
+    margin-${({ dir }) =>
+      dir === 'ltr' ? `left` : `right`}: ${GEL_SPACING_DBL};
+      &:first-child {
+        margin-${({ dir }) =>
+          dir === 'ltr' ? 'left' : 'right'}: ${GEL_SPACING_DBL};
+      }
+  }
+
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}){
+    margin-${({ dir }) =>
+      dir === 'ltr' ? `left` : `right`}: ${GEL_SPACING_DBL};
+      &:first-child {
+        margin-${({ dir }) => (dir === 'ltr' ? 'left' : 'right')}: 0;
+      }
+  }
+`;
+
+const PromoList = ({ blocks }) => {
+  const { dir } = useContext(ServiceContext);
+  const listblocks = blocks.slice(1, 4);
+
+  return (
+    <ScrollPromo dir={dir} role="list">
```

Comment:
Do we need to be explicitly setting the role here? The role [should be explicit already](https://www.w3.org/TR/html-aria/#example-redundant-role-on-list) due to `ul` element that's rendering it, unless emotion is doing some weird thing

Reply:
Spoke with @greenc05 and if I am not wrong we use role="list" in our ul tags to address a voice over bug